var/spack/repos/builtin/packages/ocamlbuild/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Ocamlbuild(MakefilePackage):
+    """OCamlbuild is a generic build tool,
+       that has built-in rules for building OCaml library and programs."""
+
+    # Add a proper url for your package's homepage here.
+    homepage = "https://ocaml.org/learn/tutorials/ocamlbuild/"
+    url      = "https://github.com/ocaml/ocamlbuild/archive/master.zip"
+    git      = "https://github.com/ocaml/ocamlbuild"
+    # list_url = "https://github.com/ocaml/ocamlbuild/releases/tag"
+    # list_url = "https://github.com/ocaml/ocamlbuild/archive/"
+
+    # Add a list of GitHub accounts to
+    # notify when the package is updated.
+    maintainers = ['gasche']
```

Comment:
This does not appear to be a valid GitHub account (for Spack).

Reply:
OK, I can ; being a github account is not enough, it has to be an account interested in spack ?

Reply:
It doesn't have to be a Spack expert, it just has to be someone who's willing to help review PRs that modify this package or investigate issues reported by users trying to build this package.

@scemama maintains our ocaml/opam packages, and may be interested in maintaining this as well.

Reply:
Hi,
OK, just put 'scemama' as a maintainer.
thanks for your work!

Reply:
OK, I will let
    maintainers = ['scemama', 'cessenat']
Now I agree with all the changes, and I've made them locally, but what should I do next ?
I tried [from https://spack.readthedocs.io/en/latest/contribution_guide.html]
git checkout develop
but it says
"error: Your local changes to the following files would be overwritten by checkout:"
Can you help please?

Reply:
You want to follow the `git add`, `git commit`, and `git push` described at https://spack.readthedocs.io/en/latest/contribution_guide.html#branching.